spec/models/user_spec.rb

```diff
@@ -1,4 +1,133 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
+  subject { described_class.new }
+
+  it 'user is valid' do
+    subject.first_name = 'John'
+    subject.last_name = 'Doe'
+    subject.email = 'johndoe@gmail.com'
+    subject.username = 'jdoe'
+    subject.phone_number = '0449 774 759'
+
```

Comment:
This should all be configured in your factory.

Reply:
As mentioned above, thanks for the clarity on this; will make sure to implement!